### PR TITLE
Update path configuration for composer-asset-plugin

### PR DESCRIPTION
**Pull request by michalwoz**
Composer install don't take "installer-paths" on the level of "config". The path which works: "config" => "fxp-asset" => "installer-paths"

Documentation

### CODE REVIEW

**Comment by Thoulah** [comment]
The default is fine, but only if you have fxp plugin install your change is required. This is already in the [docs](https://github.com/yiisoft/yii2/blob/98872c1e7dc02ae9b1c4668fb34d895481708cd5/docs/guide/structure-assets.md#using-fxpcomposer-asset-plugin).

**Comment by michalwoz** [comment]
> The default is fine, but only if you have fxp plugin install your change is required. This is already in the [docs](https://github.com/yiisoft/yii2/blob/98872c1e7dc02ae9b1c4668fb34d895481708cd5/docs/guide/structure-assets.md#using-fxpcomposer-asset-plugin).

Yes that piece of composer.json regards only the case when you use fxp plugin. But it seems that there were bug in the documentation. There was missing "fxp-asset" key. without proposed fix composer with fxp plugin installs bower to vendor/bower-asset, so either you have to change app config or create symlink vendor/bower -> vendor/bower-asset. With proposed fix composer installed bower to vendor/bower and no other steps are required.

**Comment by Thoulah** [comment]
My bad, sorry

File: docs/guide/structure-assets.md [hunk 1]
```diff
@@ -266,9 +266,11 @@ will be placed, if you want to publish them using Yii:
 
 ```json
 "config": {
-    "asset-installer-paths": {
```
Comment (samdark) [inline docs/guide/structure-assets.md]: @SilverFire, @hiqsol does that affect https://asset-packagist.org?

Comment (samdark) [reply docs/guide/structure-assets.md]: No, it won't be affected.

**Pull request merged**